pkg/capabilities/consensus/ocr3/types/encoder.go

```diff
@@ -11,3 +11,10 @@ type Encoder interface {
 }
 
 type EncoderFactory func(config *values.Map) (Encoder, error)
+
+type SignedReport struct {
+	Report     []byte
+	Context    []byte
```

Comment:
what does Context represent? please add a comment and/or change to a more meaningful name if appropriate.

should any of these be fixed length? add inline comments if there are length assumptions in other part of the code